### PR TITLE
Added a page for newcomers

### LINKED ISSUE #934: Add a documentation page for users
Funnily enough, we don't have anywhere in the documentation how to interact with the registry & Portus with docker. That is, we lack a page in which we can inform users how to login/push from the CLI, gotchas, etc. This will specially be useful for people that are new to docker and that have a Portus instance installed in their company's servers.

### PR DESCRIPTION
**Pull request by mssola**
Fixes #934

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _docs/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+on Portus). Docker provides
+[great documentation](https://www.docker.com/what-docker) for newcomers, so make
+sure to read it and come back for more information.
+
+## Docker Distribution and Portus
+
+Docker Distribution (also known as **Docker Registry**) is a storage and
+distribution solution for your Docker images. It is the backend behind the
+[Docker Hub](https://hub.docker.com/) and it's [open
+source](https://github.com/docker/distribution). The fact that is open source
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: "that it is"

File: _docs/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+means that you can have your own Docker Registry in your own servers. This is
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: "on your …"

File: _docs/first-steps.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+really interesting from lots of different reasons, but one main thing to
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: "for lots …"

File: _docs/first-steps.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+have to login again unless you explicitly log out with the `docker logout`
+command. Note that in the `logout` command you will also have to pass the
+hostname (or docker will assume Docker Hub).
+
+## Push & pull
+
+The Docker Engine will allow you to push or pull only if you are logged in. If
+this is the case, then it will be up to the authorization service (Portus in
+our case) to decide whether you can push or pull. This depends on which team
+your user belong, which role plays in said team and so on. Check out [this
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: Rephrase: "This depends on the team the user belongs to, which role he or she plays in said team and so on."

File: _docs/first-steps.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+page](/features/3_teams_namespaces_and_users.html) for more information on the
+topic.
+
+Assuming that your user is called `mssola`, you can be sure that it exists a
+namespace also called `mssola`. This is because each user has a **personal
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: "… you can be sure that a namespace with the exact same name exists."

File: _docs/first-steps.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+   you want to push this image into the **global namespace**, which is only
+   writeable by administrators. You can read more about this
+   [here](/features/3_teams_namespaces_and_users.html).
+
+Now you can push it:
+
+    $ docker push my.registry:5000/mssola/image:latest
+
+If everything goes as expected, your image should have been pushed into your
+registry! Now, whenever you want to pull the image down again, just perform the
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: I'm not sure whether or not "pull … down" is correct in this context.  I'd just simply go with "pull".

File: _docs/first-steps.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+following command:
+
+    $ docker pull my.registry:5000/mssola/image:latest
+
+# Next steps
+
+If you are also an administrator make sure to understand all the nuts and
+bolts regarding deploying and configuring Portus. In order to do this, just go
+to the [documentation page](/documentation.html). Another interesting page is
+the [features](/features.html) page, in which you will learn all the
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: IMHO better: "… learn about …"

File: _docs/first-steps.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+capabilities that Portus has to offer.
+
+Otherwise, if you have any doubt that the documentation fails to cover, don't be
+shy and ask in our [Google group](https://groups.google.com/forum/#!forum/portus-dev).
+We are eager to understand what can we do to improve your experience with
```
Comment (monstermunchkin) [inline _docs/first-steps.md]: "… what we can do …"

File: documentation.md [hunk 9]
```diff
@@ -15,21 +15,23 @@ On this page you can find information about:
 ## Quick start
 
 Are you new to Portus and you have no idea how to start with it? Please, follow
-this quick start guide. First of all, you need to install or deploy Portus. You
-can do this in multiple ways:
+this quick start guide.
 
-- Setting it up for development purposes. This is the best way to mess with
+First of all, if you are only interested to know how you can *login* and *push*
```
Comment (monstermunchkin) [inline documentation.md]: "to know" => "in knowing"

File: documentation.md [hunk 10]
```diff
@@ -15,21 +15,23 @@ On this page you can find information about:
...
+images to a registry which has Portus as an authorization service, you might be
+interested to read [this page for newcomers](/docs/first-steps.html).
```
Comment (monstermunchkin) [inline documentation.md]: "to read" => "in reading"

**Comment by monstermunchkin** [comment]
There's mixed use of "Docker" and "docker". I think we should use "Docker". Otherwise LGTM.

**Comment by mssola** [comment]
Fixed :+1:

File: _docs/first-steps.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+Docker Distribution (also known as **Docker Registry**) is a storage and
+distribution solution for your Docker images. It is the backend behind the
+[Docker Hub](https://hub.docker.com/) and it's [open
+source](https://github.com/docker/distribution). The fact that it is open source
+means that you can have your own Docker Registry on your own servers. This is
+really interesting for lots of different reasons, but one main thing to
+consider is that it delegates authorization to an "authorization service".
+
+As explained [here](http://localhost:4000/docs/How-to-setup-secure-registry.html)
```
Comment (flavio) [inline _docs/first-steps.md]: Wrong link, `http://localhost:4000` should not be hard coded

File: _docs/first-steps.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+expected, you will successfully login to your registry. Note two things:
+
+1. If you don't specify a hostname, then Docker assumes the
+   [Docker Hub](https://hub.docker.com/).
+2. You don't have to specify the `https://` part.
+
+After logging in your credentials will be stored in your system so you won't
+have to login again unless you explicitly log out with the `docker logout`
+command. Note that in the `logout` command you will also have to pass the
+hostname (or Docker will assume Docker Hub).
```
Comment (flavio) [inline _docs/first-steps.md]: Do you think it would be worth to say you have to create an account on Portus before trying to perform the login?

**Comment by flavio** [comment]
LGTM :clap:

File: _docs/first-steps.md (hunk 12 above)
Comment (mssola) [inline _docs/first-steps.md]: Makes perfect sense. Moreover, maybe it's worth it to add a new page on the different login situations: LDAP vs normal, first user admin, etc.

Comment (mssola) [inline _docs/first-steps.md]: In the end I just added a note at the beginning of the `login` section. For now this should be enough :wink:

**Comment by monstermunchkin** [comment]
LGTM :clap:

**Pull request merged**